### PR TITLE
update inperson schedule

### PR DESCRIPTION
**Pull request by rogerkuou**
Updating the inperson schedules.

### CODE REVIEW

**Comment by cpranav93** [comment]
It looks good. I think we may have to adjust as needed during the course but this is a great outline to try to stick to. We can update it further after the workshop as well. :D

**Review by cpranav93 — APPROVED** [review]

**Comment by fnattino** [comment]
Hi @rogerkuou, content wise looks great to me! 

Just wondering about the times of the coffee/lunch breaks, as these might be fixed for the in-person workshops (filter coffee needs to be prepared in advance for the breaks, sandwiches need to be delivered). So I would suggest to stick to the times that other workshops have used: 10-min coffee breaks at 10:30, 11:30, 14:30 and 15:30 and 1-hour lunch break at 12:30.

File: ds-geospatial-python/list-of-schedules.md [hunk 1]
```diff
@@ -59,25 +59,27 @@
...
-      <tr> <td>13:00</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>13:00</td>  <td>Calculating Zonal Statistics on Rasters</td> </tr>
       <tr> <td>13:45</td>  <td>Coffee break</td> </tr>
       <tr> <td>14:00</td>  <td>Parallel raster computations using Dask</td> </tr>
-      <tr> <td>14:30</td>  <td>Wrap-up</td> </tr>
-      <tr> <td>15:00</td>  <td>END</td> </tr>
+      <tr> <td>15:00</td>  <td>Coffee break</td> </tr>
+      <tr> <td>15:15</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>16:15</td>  <td>Wrap-up</td> </tr>
+      <tr> <td>16:30</td>  <td>END</td> </tr>
```
Comment (fnattino) [inline ds-geospatial-python/list-of-schedules.md:82]:
```suggestion
      <tr> <td>16:30</td>  <td>Drinks</td> </tr>
```

File: ds-geospatial-python/list-of-schedules.md [hunk 2]
```diff
@@ -59,25 +59,27 @@
...
       <tr> <td>12:00</td>  <td>Lunch break</td> </tr>
-      <tr> <td>13:00</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>13:00</td>  <td>Calculating Zonal Statistics on Rasters</td> </tr>
       <tr> <td>13:45</td>  <td>Coffee break</td> </tr>
       <tr> <td>14:00</td>  <td>Parallel raster computations using Dask</td> </tr>
-      <tr> <td>14:30</td>  <td>Wrap-up</td> </tr>
-      <tr> <td>15:00</td>  <td>END</td> </tr>
+      <tr> <td>15:00</td>  <td>Coffee break</td> </tr>
+      <tr> <td>15:15</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>16:15</td>  <td>Wrap-up</td> </tr>
```
Comment (fnattino) [inline ds-geospatial-python/list-of-schedules.md:81]:
```suggestion
      <tr> <td>16:15</td>  <td>Post-workshop survey</td> </tr>
```

File: ds-geospatial-python/schedule.md [hunk 3]
```diff
@@ -2,41 +2,36 @@
...
+      <tr> <td>11:00</td>  <td>Coffee break</td> </tr>
+      <tr> <td>11:15</td>  <td>Calculating Zonal Statistics on Rasters</td> </tr>
+      <tr> <td>12:00</td>  <td>Lunch break</td> </tr>
+      <tr> <td>13:00</td>  <td>Calculating Zonal Statistics on Rasters</td> </tr>
+      <tr> <td>13:45</td>  <td>Coffee break</td> </tr>
+      <tr> <td>14:00</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>15:00</td>  <td>Coffee break</td> </tr>
+      <tr> <td>15:15</td>  <td>Parallel raster computations using Dask</td> </tr>
+      <tr> <td>16:15</td>  <td>Wrap-up</td> </tr>
+      <tr> <td>16:30</td>  <td>END</td> </tr>
```
Comment (fnattino) [inline ds-geospatial-python/schedule.md:34]:
```suggestion
      <tr> <td>16:15</td>  <td>Post-workshop survey</td> </tr>
      <tr> <td>16:30</td>  <td>Drinks</td> </tr>
```

**Review by fnattino — CHANGES_REQUESTED** [review]
and just two minor changes for the end of the second day

**Comment by rogerkuou** [comment]
Hi @fnattino, I updated the schedule according to your comments. Can you give a final check?

**Comment by fnattino** [comment]
Thanks @rogerkuou! Should we shorten the breaks to 10 min like in the other workshops? Since we have one every hour maybe it is enough?

**Comment by rogerkuou** [comment]
> Thanks @rogerkuou! Should we shorten the breaks to 10 min like in the other workshops? Since we have one every hour maybe it is enough?

Sure will update it!

**Pull request merged**